src/blocks/SliderNew/Slider.tsx

```diff
@@ -0,0 +1,171 @@
+import Title from '../../components/Title/Title';
+import {ClassNameProps, Refable, SliderProps as SliderParams, SliderType} from '../../models';
+import {block} from '../../utils';
+
+import Arrow from './Arrow/Arrow';
+import {getSliderResponsiveParams} from './utils';
+
+import './Slider.scss';
+
+const b = block('SliderNewBlock');
+
+export interface SliderNewProps
+    extends Omit<SliderParams, 'children'>,
+        Partial<
+            Pick<
+                SwiperProps,
+                | 'onSlideChange'
+                | 'onSlideChangeTransitionStart'
+                | 'onSlideChangeTransitionEnd'
+                | 'onActiveIndexChange'
+                | 'onBreakpoint'
+            >
+        >,
+        Refable<HTMLDivElement>,
+        ClassNameProps {
+    type?: string;
+    anchorId?: string;
+    dotsClassName?: string;
+    blockClassName?: string;
+    arrowSize?: number;
+}
+
+export const SliderNewBlock = (props: PropsWithChildren<SliderNewProps>) => {
+    const {
+        animated,
+        title,
+        description,
+        type,
+        anchorId,
+        arrows = true,
+        adaptive,
+        autoplay = undefined,
+        dots = true,
+        dotsClassName,
+        disclaimer,
+        children,
+        blockClassName,
+        arrowSize,
+        onSlideChange,
+        onSlideChangeTransitionStart,
+        onSlideChangeTransitionEnd,
+        onActiveIndexChange,
+        onBreakpoint,
+    } = props;
+
+    const [slider, setSlider] = useState<SwiperClass>();
+
+    const childrenCount = React.Children.count(children);
+
+    const autoplayEnabled = useMemo(() => Boolean(autoplay), [autoplay]);
+
+    const breakpoints = useMemo(() => {
+        return getSliderResponsiveParams({
+            contentLength: childrenCount,
+            slidesToShow: props.slidesToShow,
+            mobileFullscreen: Boolean(
+                props.type && Object.values(SliderType).includes(props.type as SliderType),
+            ),
+        });
+    }, [props.slidesToShow, props.type, childrenCount]);
+
+    useEffect(() => {
+        if (!slider) return;
+        if (autoplayEnabled) {
+            slider.autoplay.start();
+        } else {
+            slider.autoplay.stop();
+        }
+    }, [slider, autoplayEnabled]);
+
+    return (
+        <div
+            className={b(
+                {
+                    'one-slide': childrenCount === 1,
+                    'only-arrows': !title?.text && !description && arrows,
+                    'without-dots': !dots,
+                    type,
+                },
+                blockClassName,
+            )}
+        >
+            {anchorId && <Anchor id={anchorId} />}
+            <Title
+                title={title}
+                subtitle={description}
+                className={b('header', {'no-description': !description})}
+            />
+            <AnimateBlock className={b('animate-slides')} animate={animated}>
```

Comment:
pagination dots over slide animation

you need some kind of solution, either show them after the end of the animation or also animate, try the options

Reply:
Dots animation added